frontend/src/components/Profile/ProfileAbout.js

```diff
@@ -0,0 +1,35 @@
+import {Button, Empty} from 'antd';
+import { Card } from 'antd';
+import {Context} from "../../store";
+import {useContext} from "react";
+
+
+const ProfileAbout = ({about, username}) => {
+
+    const [state, dispatch] = useContext(Context);
+
+    console.log(state.auth.user.username)
+
+    return (
+        <>
+        <Card title="About me" style={{marginBottom:'50px'}}>
+            {about
+                ?
+                    <p>{about}</p>
+                :
+                <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} description={"This user has nothing to say about themselves"}/>
+            }
+            {
+                (state.auth.user.roles.includes('ROLE_ADMIN') || state.auth.user.username === username)
```

Comment:
Should we let admins edit other people's profiles? (I suppose if there's inflammatory content or something but still... feels a tad weird)